todo_app/data/trello_items.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import os
+
+
+
+def get_card():
+    url = "https://api.trello.com/1/boards/N46znsdT/cards"
```

Comment:
You should try and avoid hard coding things like the board id (N46znsdT) so that other people can develop your app. You could make this an environment variable like `TrelloApi`.

This is not required for module 2, but you'll need to do it for module 3.